client/a8c-for-agencies/sections/marketplace/hosting-overview/hosting-v2/standard-agency-hosting/index.tsx

```diff
@@ -3,18 +3,41 @@ import { blockMeta, code, desktop, globe, login, reusableBlock } from '@wordpres
 import { useTranslate } from 'i18n-calypso';
 import ProfileAvatar1 from 'calypso/assets/images/a8c-for-agencies/hosting/standard-testimonial-1.png';
 import ProfileAvatar2 from 'calypso/assets/images/a8c-for-agencies/hosting/standard-testimonial-2.png';
+import { APIProductFamilyProduct } from 'calypso/state/partner-portal/types';
 import HostingAdditionalFeaturesSection from '../../../common/hosting-additional-features-section';
 import HostingFeaturesSection from '../../../common/hosting-features-section';
 import { BackgroundType1, BackgroundType2 } from '../../../common/hosting-section/backgrounds';
 import HostingTestimonialsSection from '../../../common/hosting-testimonials-section';
+import { getHostingLogo } from '../../../lib/hosting';
 import CommonHostingBenefits from '../common-hosting-benefits';
+import WPCOMPlanSelector from './wpcom-plan-selector';
 
-export default function StandardAgencyHosting() {
+import './style.scss';
+
+type Props = {
+	onAddToCart: ( plan: APIProductFamilyProduct, quantity: number ) => void;
+};
+
+export default function StandardAgencyHosting( { onAddToCart }: Props ) {
 	const translate = useTranslate();
 
 	return (
-		<div>
-			{ /* Sample Hosting sections */ }
+		<div className="standard-agency-hosting">
+			<section className="standard-agency-hosting__banner">
+				<div className="standard-agency-hosting__banner-heading">
+					<div className="standard-agency-hosting__banner-logo">
+						{ getHostingLogo( 'wpcom-hosting' ) }
+					</div>
+					<div className="standard-agency-hosting__banner-description">
+						{ translate(
+							'Optimized and hassle-free hosting for business websites, local merchants, and small online retailers.'
+						) }
+					</div>
+				</div>
```

Comment:
I think we can create a common piece for code for this. 

I went ahead and created a PR that does this: https://github.com/Automattic/wp-calypso/pull/93054

Can we merge this first and rebase it with this PR? Let me know what you think.

Reply:
I agree that handling these as a shared component is the right approach. I have the code that adds these in favor of your PR. Thanks for quickly publishing a PR. 👍